### PR TITLE
B2

### LINKED ISSUE #4: Issue2
Random `description`

```ts
console.log("Hola")
```

### PR DESCRIPTION
**Pull request by SmilingWitch**
- [x] Item1
- [x] Item2 
- [x] Close #4

### CODE REVIEW

File: file4.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function show8() {
```
Comment (TheSnakeWitcher) [inline file4.ts:1]: CAMBIA EL NOMBRE

File: file5.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+function show11() {
```
Comment (TheSnakeWitcher) [inline file5.ts:1]: CAMBIA EL NOMBRE

**Review by TheSnakeWitcher — CHANGES_REQUESTED** [review]
Pa que cambies esa historia 😄

File: file4.ts (hunk 1 above)
Comment (SmilingWitch) [reply file4.ts]: Fixed in [16f2105](https://github.com/TheSnakeWitcher/nena-training/pull/5/commits/16f2105ccd0f2f262b8355d681d2a0a33693bdc7)

File: file5.ts (hunk 2 above)
Comment (SmilingWitch) [reply file5.ts]: Fixed in [2dabb81](https://github.com/TheSnakeWitcher/nena-training/pull/5/commits/2dabb814251a63e034e7cec296ac5d7dddece72c)

**Pull request merged**